### DISCUSSION TITLE
'Unknown org' error for organization with uppercase name and private repositories

### DISCUSSION THREAD

**Discussion by Vidzhel** [community/community#15448] [Other Feature Feedback, Questions, & Ideas]
Hello there, It seems that I'm unable to search code in one of the organization I'm a member of. I've tries uppercase and lowercase version of the name but it says that the organization doesn't exist at all, also tried with `user:` instead of `org:`. I literally just copied the name from the url.

![uppercase version of name](https://i.imgur.com/gOwPR9C.jpeg)
![lowercase](https://i.imgur.com/5U3ycXU.jpeg)
![i'm a member of the org](https://i.imgur.com/owa98Jh.jpeg)

**Comment by Nezteb** [comment]
I have this same problem with another org that ALSO has a `-` (dash/hyphen) in the name. I assume that is the bug.

**Reply by look** [reply]
@Nezteb If it is OK, can you post the name of the organization so I can look into it as well?

**Reply by Nezteb** [reply]
I just saw your other reply about the org needing to be part of the technology preview in order for the new CS feature to work; I too was confused because the last two orgs I was part of must have been part of it without me realizing!

Org is https://github.com/players-lounge 😄💙

**Reply by look** [reply]
I've added this org. We index the orgs you belong to when you're onboarded, but I think we're missing orgs you join after that.

**Comment by look** [comment]
@Vidzhel I checked and the organization wasn't part of the Technology Preview. I added it. Did you get added to this organization after joining the Code Search technology preview?

**Reply by Vidzhel** [reply]
Cool, thanks. I was thinking that's the case but I had been a part of 2 private organisations before joining the tech preview, so thought that it may be something different

**Reply by janhartigan** [reply]
@look is there some automatic way to do this? I just joined [a new org](https://github.com/goartica) and I'm running into the same issue.

**Reply by look** [reply]
@janhartigan if you have one of your new colleagues join the Technology Preview, the organization will get indexed (at this time, we clear the waitlist daily). We haven't had time to prioritize making this automatic, but we are working to index more repositories in the new code search index so that it doesn't happen any more.
